README.md

```diff
@@ -16,7 +16,9 @@ https://github.com/ManarOmari/Portoflio/tree/main
 
 ## Getting Started
 
-
+started to buils mobile sketon for a Protoflio template 
+first : header contains a logo and tool bar 
+then set background image for the content section which conains a text and images 
```

Comment:
- Please add the CSS styling to your project as it is missing that, making your project not look like the Figma design 
##### Figma design
![Screenshot from 2022-05-27 06-46-53](https://user-images.githubusercontent.com/58771507/170638044-2c16f8ae-e943-474e-b99b-a34a56e368da.png)

##### Your design
![Screenshot from 2022-05-27 06-43-19](https://user-images.githubusercontent.com/58771507/170638079-2b5d3f75-3797-4d9d-981e-f9cc7404171f.png)

**Notes:**
- Please also make sure you are done with the given sectioned task, before requesting a review
- Please also make sure you use only one branch when neccessary

If you need for explanation concerning this feedback, Feel free to reach out to me on Slack: `@Kanu mike chibundu`